wanaprez/prez.js

```diff
@@ -186,6 +200,9 @@ function _doPrez({rootNode, slides, title}) {
             case KEY_RIGHT:
                 _goNext();
                 break;
+			case KEY_F:
+				_toggleFullscreen();
+				break;
```

Comment:
Please use 4 spaces to indent the code, not tabs
```suggestion
            case KEY_F:
                _toggleFullscreen();
                break;
```